glean-core/tests/labeled.rs

```diff
@@ -0,0 +1,193 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+mod common;
+use crate::common::*;
+
+use serde_json::json;
+
+use glean_core::metrics::*;
+use glean_core::storage::StorageManager;
+use glean_core::{CommonMetricData, Lifetime};
+
+#[test]
+fn can_create_labeled_metric() {
```

Comment:
Do we need one of this test for each supported metric type?

Reply:
Yeah, kind of. Though we also test this from the Kotlin side, where we already have tests for most sub-metric types

Reply:
I'd still test it here. Let's think of the Kotlin tests as a plus, given that this core code would be used by iOS specific code too (and desktop!) in the future.

Reply:
This slipped through the cracks :)